register/register_windows.go

```diff
@@ -0,0 +1,8 @@
+//+build windows
```

Comment:
Nit: no need to add this line since the file name *_windows already defines the context where it should be built